src/imp/tracer_imp.js

```diff
@@ -883,11 +883,13 @@ export default class TracerImp extends EventEmitter {
         let clockOffsetMicros = this._clockState.offsetMicros();
 
         // Diagnostic information on the clock correction
-        this.logStable('cr/time_correction_state', {
-            offset_micros  : clockOffsetMicros,
-            active_samples : this._clockState.activeSampleCount(),
-            ready          : clockReady,
-        });
+        if (this._options.debug) {
```

Comment:
Both my original change and your suggestion (which LGTM) cause the

```
        1) should flush at a regular interval
```

test to fail... I was seeing rather strange/confounding stuff in my debugging of the test failure and eventually realized that `mocha` runs the tests in parallel yet some test cases (e.g., this one) expect to be bound to a specific `Tracer` instance.

I fixed this but I'm not happy with the solution... I'll add some comments inline to explain.